nservicebus/azure-service-bus/send-atomic-with-receive-note.include.md

```diff
@@ -0,0 +1 @@
+Note: `SendAtomicWithReceive` level is only supported in Versions 7 and above of the Azure Service Bus transport. It requires destination and receive queues to be in the same namespace.
```

Comment:
Isn't this supposed to be "TransportTransactionMode.SendsAtomicWithReceive", i.e. Sends and not SendAtomicWithReceive?